chapters/annotations.tex

```diff
@@ -103,16 +103,17 @@ \subsection{Annotations for Figures}\doublelabel{annotations-for-figures}
 \autoref{plot-curves}, that all share a common x and y axis with properties
 described in \autoref{axis-properties}.
 
-The \lstinline!title! of a \lstinline!Figure! shall be non-empty and is
-mandatory, as every figure needs a title for presentation in contexts such as a
-list of figures, and as it is generally hard for tools to automatically generate
-a meaningful title.  On the other hand, when the \lstinline!title! of a
-\lstinline!Plot! isn't provided, the tool produces a default, but the default is
-allowed to be empty.  Providing the empty string as \lstinline!title! of a
-\lstinline!Plot! means that no title should be shown.  The plot title is not to
-be confused with the plot \emph{label} which is never empty, see below. Variable
-replacements, as described in \autoref{variable-replacements}, can be used in the
-\lstinline!title! of \lstinline!Figure! and \lstinline!Plot!.
+Both \lstinline!Figure! and \lstinline!Plot! can have an optional title.
+Providing an empty string, or leaving the default of an empty string, for a
```

Comment:
I think the pseudo code is usually enough to convey default behavior in a case like this.  Hence, one can simply write:
> When the `Figure` `title` is the empty string (the default), the tool must…